.github/fabricbot.json

```diff
@@ -1,965 +0,0 @@
-{
-  "version": "1.0",
-  "tasks": [
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssueCommentResponder",
-      "version": "1.0",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isOpen",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "name": "isAction",
-              "parameters": {
-                "action": "created"
-              }
-            },
-            {
-              "name": "hasLabel",
-              "parameters": {
-                "label": "Status: No Recent Activity"
-              }
-            },
-            {
-              "name": "hasLabel",
-              "parameters": {
-                "label": "Needs: Author Feedback"
-              }
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "noActivitySince",
-                  "parameters": {
-                    "days": 7
-                  }
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isCloseAndComment",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "name": "isActivitySender",
-              "parameters": {
-                "user": {
-                  "type": "author"
-                }
-              }
-            },
-            {
-              "name": "activitySenderHasPermissions",
-              "parameters": {
-                "permissions": "none"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issue_comment"
-        ],
-        "taskName": "For issues closed due to inactivity, re-open an issue if issue author posts a reply within 7 days.",
```

Comment:
@mkArtakMSFT @wtgodbe This task did not get ported because it relies on the `noActivitySince` filter.